app/server/functions/src/main/kotlin/com/supercilex/robotscouter/server/functions/Templates.kt

```diff
@@ -50,32 +50,38 @@ fun matchTemplateMetrics() = metrics {
     header("a", "Scout info")
     text("b", "Name")
 
-    header("c", "Auto")
-    checkbox("d", "Crossed baseline")
-    selector("e", "Put cube in Switch") {
-        +Item("a", "Didn't attempt")
-        +Item("b", "Attempted")
-        +Item("c", "Successful")
-        +Item("d", "Unknown", true)
+    header("c", "Sandstorm")
+    selector("d", "Starting location") {
+        +Item("a", "HAB Level 2")
+        +Item("b", "HAB Level 1")
+        +Item("c", "Unknown", true)
     }
-    selector("f", "Put cube in Scale") {
-        +Item("a", "Didn't attempt")
-        +Item("b", "Attempted")
-        +Item("c", "Successful")
-        +Item("d", "Unknown", true)
+    checkbox("e", "Successfully crossed HAB line")
+    counter("f", "Panels on Cargo Ship")
+    counter("g", "Panels on low Rocket Hatches")
+    counter("h", "Panels on middle/high Rocket Hatches")
+    counter("i", "Cargo in Cargo Ship")
+    counter("j", "Cargo in low Rocket Bays")
+    counter("k", "Cargo in middle/high Rocket Bays")
```

Comment:
Do you think teams will be trying to deliver both panels _and_ cargo? Or will they tend towards one or the other. If it's the former, I think we should group it by destination, not delivery type. If it's the latter, then all good. 👍

Reply:
I think there will definitely be some teams that will deliver both panels and cargo during a match but many others would likely just focus on one game element. I like grouping things by destination though I'm not too sure how we would keep track of what game piece they deliver to those destinations.

Reply:
Sounds good. 👍